src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java

```diff
@@ -1997,6 +1994,12 @@ public Component getItemName(ItemStack stack) {
                     new CoverPlaceBehavior(GTCovers.ITEM_FILTER)))
             .onRegister(materialInfo(new ItemMaterialInfo(new MaterialStack(GTMaterials.Zinc, GTValues.M * 2))))
             .onRegister(compassNode(GTCompassSections.COVERS, GTCompassNodes.COVER)).register();
+    public static ItemEntry<ComponentItem> SMART_ITEM_FILTER = REGISTRATE
+            .item("smart_item_filter", ComponentItem::create)
+            .lang("Smart Item Filter")
+            .onRegister(attach(new ItemFilterBehaviour(SmartItemFilter::loadFilter), new CoverPlaceBehavior(GTCovers.ITEM_FILTER)))
+            .onRegister(materialInfo(new ItemMaterialInfo(new MaterialStack(GTMaterials.Zinc, GTValues.M * 2))))
```

Comment:
isn't this zinc positive since you only use 6 foil for it?

Reply:
i have no clue how the item builder works i just copied it from the item filter 😭

Reply:
Set it to like M*3/2